litani

```diff
@@ -537,13 +554,32 @@ def init(args):
     except KeyError:
         run_id = str(uuid.uuid4())
 
-    cache_dir = pathlib.Path(
-        tempfile.gettempdir()) / "litani" / "runs" / run_id
-    cache_dir.mkdir(parents=True)
+    if args.output_prefix:
+        output_prefix = pathlib.Path(args.output_prefix).resolve()
+    else:
+        output_prefix = pathlib.Path(tempfile.gettempdir())
+
+    if args.output_directory:
```

Comment:
Since `--output-prefix` and `--output-directory` are mutually exclusive, it would be good to reflect that here. Especially since if `--output-directory` is set, you're still assigning the `output_prefix` variable on line 560 and never using it, it just makes the code less obvious to read.

```python
if args.output_directory:
    cache_dir = pathlib.Path(args.output_directory).resolve()
else:
  if args.output_prefix:
      output_prefix = pathlib.Path(args.output_prefix).resolve()
  else:
      output_prefix = pathlib.Path(tempfile.gettempdir())
  cache_dir = output_prefix / "litani" / "runs" / run_id
```